release-team/role-handbooks/communications/README.md

```diff
@@ -47,81 +47,183 @@ There is a channel on the Kubernetes Slack workspace, `release-comms`, which is
 
 ## Release Milestone Activities
 
-### Week 1
+<table>
+    <tr>
+        <td><h3><b>Release Week</h3></b></td>
+        <td><h3><b>RW 1.19</h3></b></td>
```

Comment:
I think it might be better to just drop this column and update the actual release week (and then we can do this every release). 

1.20 will probably also have a non-standard release cycle, so who ever leads comms will probably need to update this column (and rename it) for 1.20..what if we just update the calendar every release and take into consideration that future release cadence might change.  Make it a week 1 task for future releases as well.

Reply:
also thought about it, but we intended to make the comms more clear and transparent. Therefore I think it would be beneficial to have the 13-week cycle as a reference, as we drove the last release very well with it. 

Just don't want to cause too many actions, that in 2021 the comms team needs to figure out at which time they have to communicate, publish and trigger.

Reply:
Do you feel good on that front, @jeremyrickard? If so, we can merge this in.

Reply:
Sorry! I thought I had replied here.  2021 comms teams can always revisit.